src/DataGrid.tsx

```diff
@@ -23,7 +23,7 @@ import {
   RowExpandToggleEvent,
   RowGetter,
   SelectedRange,
-  CustomRowRendererProps,
+  IRowRendererProps,
```

Comment:
Should we rename it to `RowRendererProps` instead as we not using the `I` prefix for any other interface?

Reply:
It conflicts with `RowRenderer`'s `RowRendererProps` unfortunately.